docs/relationships.md

```diff
@@ -42,59 +52,50 @@ const client = new CozyClient({
 })
 ```
 
-With the help of the schema, Cozy-Client will be able to manage the `relationships` attribute of your documents for you.
+We explain below what are the different types of relations.
 
 ### Relation types
 
-Cozy-Client [predefines](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/associations/helpers.js) four basic relation types: 
+Cozy-Client [predefines](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/associations/helpers.js) two basic relation types that must be specified in the schema:
 
-- `'has-one'`
-- `'has-many'`
-- `'has-one-in-place'`
-- `'has-many-in-place'`
+- `'has-one'`: the relation has a unique reference.
+- `'has-many'`: the relation can have several references.
 
-For new doctypes you should use `'has-one'` or `'has-many'`. They use your `relationships` attribute and link either a single document or an array of documents.
 
-```javascript
-const schema = {
-  todos: {
-    doctype: 'io.cozy.books',
-    attributes: {},
-    relationships: {
-      authors: {
-        doctype: 'io.cozy.contact',
-        type: 'has-many'
-      }
-    }
-  }
-}
-```
+#### Files relations
+
+The files implements a special type of relation: `'io.cozy.files:has-many'`. This relation name is `referenced_by`, which is an array of references, just like the `'has-many'` relation. 
+
+The stack implements routes to handle this kind of relation on the `/files` endpoint. See the [stack documentation](https://docs.cozy.io/en/cozy-stack/references-docs-in-vfs/).
+
+Note specific [view index](https://docs.cozy.io/en/tutorials/data/queries/#usage-example-references) are defined on `referenced_by` relationships, allowing fast queries on it.
 
-```javascript
-const document = {
-  "_id": "mobydick",
-  "relationships": {
-    "authors": {
-      "data": [
-        { "_id": "hermanmelville", "_type": "io.cozy.contacts" }
-      ]
-    }
-  }
-}
-// there should be an io.cozy.contact document 
-// for which the _id is "hermanmelville"
-```
 
-`'has-one-in-place'` or `'has-many-in-place'` are here for backward compatibility. Instead of using the `relationships` attribute, you reference directly the ids of linked documents at the root of your data:
+#### Customize relations
+
+You are free to create your own relation type if you need a special behaviour.
+
+For instance, [Banks doctypes](https://github.com/cozy/cozy-banks/blob/master/src/doctypes.js) implements `HasManyBills` or `HasManyReimbursements` that inherits from `HasManyInPlace`. 
```

Comment:
Here we refer to `HasManyInPlace` for the first time without defining what it is, maybe we should swap this section with the `Old relation type` so it will be defined first?